charts/gardener-extension-shoot-rsyslog-relp-admission/Chart.yaml

```diff
@@ -1,5 +1,10 @@
+# SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
 apiVersion: v1
 appVersion: "1.0"
 description: A Helm chart for the Gardener Shoot Rsyslog Relp admission controller
-name: gardener-extension-shoot-rsyslog-relp
+name: gardener-extension-shoot-rsyslog-relp-admission
```

Comment:
I was wondering whether this is a breaking change? If someone already had the chart deployed with help, does he/she need something to do to switch to the new name?

Reply:
Good point, I think I tried upgrading it in the local setup with skaffold - first ran `make extensions-up` with the old release name, then `make extensions-up` with the new name, however let me try it out again and perform some more checks.

Reply:
The only difference should be the chart's name when the `runtime` and `application` subcharts are installed in the same cluster. In that case one would usually install the parent chart directly. Helm handles the rename of the charts without a problem and since `.Chart.Name` is not used anywhere and there are no chart dependencies there should be no additional work that has to be done.

Reply:
I anyway added a release note about this change.